src/include/daos_types.h

```diff
@@ -279,6 +279,19 @@ typedef struct {
 	struct daos_rebuild_status	pi_rebuild_st;
 } daos_pool_info_t;
 
+/*
+ * DAOS management pool information
+ */
+typedef struct {
```

Comment:
(style)  do not add new typedefs

Reply:
I would like to disregard this style warning in this patch because the newly introduced typedef / struct is in the same style as the other definitions in the file.